snapcraft/internal/repo/_base.py

```diff
@@ -223,13 +193,14 @@ def normalize(self, unpackdir: str) -> None:
 
         :param str unpackdir: directory where files where unpacked.
         """
-        self._remove_useless_files(unpackdir)
-        self._fix_artifacts(unpackdir)
-        self._fix_xml_tools(unpackdir)
-        self._fix_shebangs(unpackdir)
+        cls._remove_useless_files(unpackdir)
+        cls._fix_artifacts(unpackdir)
+        cls._fix_xml_tools(unpackdir)
+        cls._fix_shebangs(unpackdir)
 
+    @classmethod
```

Comment:
maybe
```suggestion
    @staticmethod
```

and drop cls